app/models/user.rb

```diff
@@ -422,6 +426,16 @@ def va_profile_v2_email
     vaprofile_contact_info&.email&.email_address
   end
 
+  def va_profile_phone
+    home = vaprofile_contact_info&.home_phone
+    home.area_code + home.phone_number
+  end
+
+  def va_profile_mobile_phone
+    mobile = vaprofile_contact_info&.mobile_phone
+    mobile&.area_code.to_s + mobile&.phone_number.to_s
```

Comment:
I would probably do an early return for `vaprofile_contact_info&.mobile_phone`, then you won't have to worry about the safe navigation operators. Cause otherwise you have to chain them the whole way down. For example, right now this will break if `mobile` is `nil`, because there isn't a safe navigation operator on the `.to_s` part of these

Reply:
Actually I lied, `nil.to_s` works, so

Reply:
Thank you for verifying.

Reply:
This code review is resolved. Those code has been changed since then